### ISSUE TITLE
Node.js  Build WorkGroup Meeting 2022-08-24

### ISSUE THREAD

**Issue by mhdawson** [nodejs/build#3015]
## Time

**UTC Wed 24-Aug-2022 22:00 (10:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Wed 24-Aug-2022 15:00 (03:00 PM) |
| US / Mountain | Wed 24-Aug-2022 16:00 (04:00 PM) |
| US / Central  | Wed 24-Aug-2022 17:00 (05:00 PM) |
| US / Eastern  | Wed 24-Aug-2022 18:00 (06:00 PM) |
| EU / Western  | Wed 24-Aug-2022 23:00 (11:00 PM) |
| EU / Central  | Thu 25-Aug-2022 00:00 (12:00 AM) |
| EU / Eastern  | Thu 25-Aug-2022 01:00 (01:00 AM) |
| Moscow        | Thu 25-Aug-2022 01:00 (01:00 AM) |
| Chennai       | Thu 25-Aug-2022 03:30 (03:30 AM) |
| Hangzhou      | Thu 25-Aug-2022 06:00 (06:00 AM) |
| Tokyo         | Thu 25-Aug-2022 07:00 (07:00 AM) |
| Sydney        | Thu 25-Aug-2022 08:00 (08:00 AM) |

Or in your local time:
* https://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Build%20WorkGroup+Meeting+2022-08-24&iso=20220824T2200
* or https://www.wolframalpha.com/input/?i=10PM+UTC%2C+Aug+24%2C+2022+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1vrK_4AhKJduz_hZWgbyFOqjKOyuln8X1EuYmKAZiDbM/edit>

## Agenda

Extracted from **build-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/build

* Support for llnode builds [#3017](https://github.com/nodejs/build/issues/3017)
* Onboarding F3n67u to the build wg [#3003](https://github.com/nodejs/build/issues/3003)
* Jenkins will require Java 11 from September LTS release [#2984](https://github.com/nodejs/build/issues/2984)
* Interest in establishing a "Build Helper" role [#2550](https://github.com/nodejs/build/issues/2550)
* Official Windows arm64 binaries [#2540](https://github.com/nodejs/build/issues/2540)

## Invited

* Build team: @nodejs/build

## Notes

The agenda comes from issues labelled with `build-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

* link for participants: <https://zoom.us/j/715960833>
* For those who just want to watch: We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at <https://www.youtube.com/c/nodejs+foundation/live> when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.
* youtube admin page: <https://www.youtube.com/my_live_events?filter=scheduled>

**Comment by dennisameling** [comment]
Hi @mhdawson, I could join this meeting for the Windows arm64 part if that's of interest. I can't commit to long-term support right now, but am trying to contribute in several areas to at least get the builds to work. Otherwise happy to just listen in through YouTube 👍🏼

**Comment by sxa** [comment]
On the basis that
- I have done very little on the build side in the last few weeks due to release commitments elsewhere
- This time is 2300 in the UK, midnight for most of the EU, so not very convenient
- Rod can't make it this time

I propose cancelling. I do need to follow up on some of the activities relating to Windows/aarch64 but that can be done outside a build call between the interested parties :-)

**Comment by sxa** [comment]
@dennisameling If you're not on the Node slack feel free to join us on there for more discussions - looks like you're not part of the Linaro team so it would be interesting to know what your interest is in Windows/aarch64. Any help is welcome :-)

**Comment by mhdawson** [comment]
+ 1 to cancel based on suggestion from @sxa

**Comment by nsait-linaro** [comment]
Yes, Linaro folks are also on holiday. I suggest we schedule a meeting after September 6th for Windows/Arm64 discussions.

Thanks to @pbo-linaro! We have the test CI working with all tests passing - https://ci.nodejs.org/job/windows-arm-simple/. 

@sxa had concerns about running the tests with admin privilege. We need to sort that out. And figure out how to deploy the machines and CI configuration to the official CI job - https://ci.nodejs.org/job/node-test-binary-windows-js-suites/

Thanks, @dennisameling for looking at the arm64 native toolchain support for node. That would be helpful to reduce the build time significantly.

**Comment by richardlau** [comment]
Sounds like everyone is 👍  to cancelling.

**Comment by mhdawson** [comment]
Cancelling.
